server/server.py

```
@@ -349,6 +624,8 @@ def main() -> None:
     print(f"  static dir : {STATIC_DIR}")
     print(f"  websocket  : /ws")
     print(f"  score api  : /scores/{{game_type}}")
+    print(f"  admin api  : /api/admin/vitals  (header: Authorization: Bearer <ADMIN_PASSWORD>)")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)